sw/device/lib/dif/dif_entropy.c

```diff
@@ -93,12 +93,40 @@ dif_entropy_result_t dif_entropy_configure(const dif_entropy_t *entropy,
   return kDifEntropyOk;
 }
 
+inline static bool get_es_valid(const dif_entropy_t *entropy) {
+  uint32_t reg = mmio_region_read32(entropy->params.base_addr,
+                                    ENTROPY_SRC_INTR_STATE_REG_OFFSET);
+  return bitfield_bit32_read(reg, ENTROPY_SRC_INTR_STATE_ES_ENTROPY_VALID_BIT);
```

Comment:
nit: you can probably use `mmio_region_get_bit32()`

Reply:
updated.